bitnami/kafka/templates/statefulset.yaml

```diff
@@ -177,7 +177,7 @@ spec:
                   fieldPath: metadata.name
             - name: KAFKA_CFG_ZOOKEEPER_CONNECT
               {{- if .Values.zookeeper.enabled }}
-              value: {{ include "kafka.zookeeper.fullname" . | quote }}
+              value: "{{ include "kafka.zookeeper.fullname" . }}{{ printf "%s" (tpl .Values.zookeeperChrootPath .) }}"
```

Comment:
take into account that this is an URL. The port is missing in this way and will probably be appended after the added endpoint.
Also, use the `printf` to render the full string instead of concatenating both directly.

Reply:
Current chart don't use port in this string. I can add it in other PR.

printf for full string is committed. Or better to move this logic in function?